upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -141,3 +141,6 @@ There you can find links to upgrade notes for other versions too.
 
 - made tests to be domain independent ([#2051](https://github.com/shopsys/shopsys/pull/2051))
     - see #project-base-diff to update your project
+
+- made parameters overridable by ENV variables ([#2055](https://github.com/shopsys/shopsys/pull/2055))
+    - see #project-base-diff to update your project
```

Comment:
I think a little info about where set the values instead of parameters.yaml file will be nice. Maybe a link to the documentation.

Reply:
fixed